source/includes/wp-api-v3/_products.md

```diff
@@ -1481,7 +1481,11 @@ woocommerce.get("products").parsed_response
 | `parent_exclude`  | array   | Limit result set to all items except those of a particular parent ID.                                                                                                     |
 | `slug`            | string  | Limit result set to products with a specific slug.                                                                                                                        |
 | `status`          | string  | Limit result set to products assigned a specific status. Options: `any`, `draft`, `pending`, `private` and `publish`. Default is `any`.                                   |
-| `type`            | string  | Limit result set to products assigned a specific type. Options: `simple`, `grouped`, `external` and `variable`.                                                           |
+| `include_status`  | string  | Limit result set to products with any of the specified statuses. Multiple statuses can be provided as a comma-separated list. Options: `any`, `trash`, `draft`, `pending`, `private`, and `publish`.|
+| `exclude_status`  | string  | Exclude products with any of the statuses from result set. Multiple statuses can be provided as a comma-separated list. Options: `trash`, `draft`, `pending`, `private`, and `publish`.|
```

Comment:
```suggestion
| `exclude_status`  | string  | Exclude products from result set with any of the specified statuses. Multiple statuses can be provided as a comma-separated list. Options: `trash`, `draft`, `pending`, `private`, and `publish`.|
```

Reply:
Also might be worth noting that `include_status` overrides `status` and `exclude_status` overrides `include_status` (is that right?)

Same would apply for the`type` params.

Reply:
Thanks! I pushed the change, could you have another look? 🙏